doc/user-guides/acrn-dm-parameters.rst

```diff
@@ -107,6 +115,22 @@ Here are descriptions for each of these ``acrn-dm`` command line parameters:
           -i /run/acrn/ioc_$vm_name,0x20
           -l com2,/run/acrn/ioc_$vm_name
 
+   * - :kbd:`--intr_monitor`
+     - Enable interrupt strom monitor for UOS. If user want to prevent the interrupt 
+       storm from UOS, this option should be used.
+       
+       Example:: 
+
+         intr_storm_monitor="--intr_monitor 10000,10,1,100"
+
+       - ``10000``: interrupt rate larger than 10000/s will be treated as interrupt 
```

Comment:
Can you introduce in the list of descriptions, the same wording that is used in the help, i.e. `threshold/s,probe-period(s),delay_time(ms),delay_duration(ms)`. That would help the reader map these more easily.